R/html-tweak.R

```diff
@@ -277,6 +277,9 @@ badges_extract_text <- function(x) {
   badges_extract(xml)
 }
 
+find_badges_paragraph <- function(html){
```

Comment:
Why is this a separate function?

Reply:
I guess I thought it'd be bigger, but since it isn't, I changed this. :-)